File: Professional Image Modifier 5/[VB]-Professional Image Modifier 5/VB/ImageFunctions/Forms/FrmProcessing.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace ImageFunctions.Forms
	Partial Public Class FrmProcessing
		Inherits Form

		Public Property description() As String

		Public Sub New(ByVal description As String)
			InitializeComponent()
			lblDescription.Text = description

		End Sub

		' Allows user to close the Processing Information form!
		Private Sub closeToolStripMenuItem_Click(ByVal sender As Object, ByVal e As EventArgs) Handles closeToolStripMenuItem.Click
			Me.Close()
		End Sub


	End Class
End Namespace

File: Professional Image Modifier 4/[C#]-Professional Image Modifier 4/C#/ImageFunctions/Forms/FrmProcessing.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ImageFunctions.Forms
{
	public partial class FrmProcessing : Form
	{
		public string description { get; set; }

		public FrmProcessing(string description)
		{
			InitializeComponent();
			lblDescription.Text = description;

		}

		// Allows user to close the Processing Information form!
		private void closeToolStripMenuItem_Click(object sender, EventArgs e)
		{
			this.Close();
		}


	}
}
